data/syntax-highlighting/vim/syntax/meson.vim

```diff
@@ -150,7 +150,7 @@ hi def link mesonEscape		Special
 hi def link mesonNumber		Number
 hi def link mesonBuiltin	Function
 hi def link mesonBoolean	Boolean
-if exists("meson_space_error_higlight")
```

Comment:
I believe this will change how vim users experience things, as this is the only instance of this, whereas the correct spelling has a couple of instances.

This _doesn't_ _appear_ to be a backwards compatibility shim, although it could be. If it is, I'd be happy to change it to include a comment calling this out...